README.md

```diff
@@ -1,2 +1,29 @@
-# BITEhack2024
-GradientAbsinthe teams repo for BITEhack 2024 hackathon
+# BITEhack2024 - Quit It
+Welcome to **Quit It**! - an innovative application designed to support individuals in overcoming their habits with the help of friends or aiding their loved ones in their journey to break free from addiction. With a unique feature that seamlessly integrates community support and the proximity of your closest allies, **Quit It**! provides an effective platform for both you and your friends to conquer habits together.
+
+## Introduction
+Sarah - a determined individual who had decided it was time to break free from the chains of her addiction. Opting not to face this journey alone, she turned to **Quit It**! an app that fostered communal support. Sarah invited her closest friends to join her on this challenging path to quit the habit.
+
+As Sarah and her friends embarked on this collective journey, they found strength in each other's company. She knew that she had her friends on her side, allowing them to share her triumphs, struggles, and milestones. The camaraderie formed a powerful support network that [bolstered](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3917180/pdf/IJNMR-19-19.pdf) her resolve!
+
+Unlike previous attempts to quit in isolation, Sarah noticed a significant difference this time. The support of her friends and shared milestones among them accelerated the process of overcoming her individual addictions. The determination fueled a positive and competitive spirit, turning the challenging endeavor into a shared triumph.
+
+**Quit It**! not only became a tool for tracking progress but also for celebrating each milestone achieved. Through the power of friendship and mutual support, Sarah discovered that quitting a habit didn't have to be a solitary struggle. With **Quit It**, she experienced firsthand how facing the journey together not only made it more manageable but also transformed it into a victory.
```

Comment:
XD

Po prostu wyjaśnij dlaczego to pomaga i dodaj linki do artykułów. To jest cringe AF

Reply:
Sam jesteś cringe af. Naucz się robić quality review.
![review](https://github.com/WojciechBarczynski/BITEhack2024/assets/79122265/72d5779d-fdee-4406-b789-589706f258b4)